src/apps/loyalty/server/template.tsx

```diff
@@ -35,6 +35,7 @@ export default ({ styles, html, entrypoint, bootstrapData, sharify, baseURL }: T
           ${bootstrapData ? "<script>" + bootstrapData + "</script>" : ""}
           <script src="${baseURL}/bundles/commons.chunk.js"></script>
           <script src="${entrypoint}"></script>
+          <script src="${baseURL}/bundles/analytics.js"></script>
```

Comment:
I _guess_ @orta is getting at refactoring the template to take a list of scripts to include. I’d say, let’s wait till we need the situation where a script only needs to be injected in _some_ pages. For now this list is the same for all pages.